arm/Microsoft.Network/privateDnsZones/AAAA/readme.md

```diff
@@ -5,20 +5,17 @@ This module deploys a Private DNS Zone AAAA record.
 ## Navigation
 
 - [Resource Types](#Resource-Types)
-- [Parameters](#Parameters)
 - [Outputs](#Outputs)
+- [Parameters](#Parameters)
```

Comment:
@MrMCake has this been regenerated with the latest script? I see some glitches on my end

Reply:
Glitch no more